apps/catalog/catalog/src/app/marketplace/title/list/list.component.html

```diff
@@ -112,7 +112,7 @@ <h1 class="mat-display-2 page-title">Library</h1>
 
   <!-- TITLE CARD -->
   <ng-template listPageCard let-title>
-    <movie-card [movie]="title" [link]="['/c/o/marketplace/title',  title.objectID]" size="poster" [attr.test-id]="'movie-card_' + title.objectID">
+    <movie-card [movie]="title" [link]="title.objectID" [queryParams]="availsForm.valid ? availsForm.value : null" size="poster" [attr.test-id]="'movie-card_' + title.objectID">
```

Comment:
lets keep it as an array
```suggestion
    <movie-card [movie]="title" [link]="['/c/o/marketplace/title', title.objectID]" [queryParams]="availsForm.valid ? availsForm.value : null" size="poster" [attr.test-id]="'movie-card_' + title.objectID">
```